docs/faq.rst

```diff
@@ -251,6 +251,25 @@ using a Read the Docs :doc:`config-file/index`.
 .. _pip requirements file: https://pip.pypa.io/en/stable/user_guide.html#requirements-files
 
 
+Can I install into a environment with pinned versions?
+------------------------------------------------------
+To ensure proper installation of the python package, the ``pip`` :ref:`install
+method <config-file/v2:python.install>` will automatically upgrade every
+dependency to its most recent version. If instead you'd like to pin your
```

Comment:
```suggestion
dependency to its most recent version in case they aren't pinned. If instead you'd like to pin your
```

Reply:
maybe something like this?
```suggestion
dependency to its most recent version in case they aren't pinned by the package definition. If instead you'd like to pin your
```

Reply:
yeah, that sounds better